src/timeseriesinsights/README.md

```diff
@@ -1,95 +1,129 @@
-Microsoft Azure CLI 'timeseriesinsights' Extension
-==========================================
+# Azure CLI timeseriesinsights Extension #
+This is the extension for timeseriesinsights
 
-This package is for the 'timeseriesinsights' extension:
-
-```sh
-az timeseriesinsights
+### How to use ###
+Install this extension using the below CLI command
 ```
-
-## Install
-
-```sh
 az extension add --name timeseriesinsights
 ```
 
-## Uninstall
-
-```sh
-az extension remove --name timeseriesinsights
+### Included Features ###
+#### timeseriesinsights environment ####
+##### Gen1 Create #####
 ```
-
-## Examples
-
-All commands are for Bash. For PowerShell, please set the variables like 
-
-```powershell
-$rg='rg1'
+az timeseriesinsights environment gen1 create --name "env1" --location westus --data-retention-time "P31D" \
```

Comment:
I remember that in a previous email the new design is to use abbreviations to avoid long names, like

```
az tsi env gen1 create
```